index.bs

```diff
@@ -55,11 +55,17 @@ urlPrefix: https://tc39.es/ecma262/; spec: ECMASCRIPT
   text: IsDetachedBuffer; url: #sec-isdetachedbuffer
   text: IsInteger; url: #sec-isinteger
   text: OrdinaryObjectCreate; url: #sec-ordinaryobjectcreate
+  text: SameValue; url: #sec-samevalue
   text: SetFunctionLength; url: #sec-setfunctionlength
   text: SetFunctionName; url: #sec-setfunctionname
   text: Type; url: #sec-ecmascript-data-types-and-values
  text: TypeError; url: #sec-native-error-types-used-in-this-standard-typeerror; type: exception
  text: map; url: #sec-array.prototype.map; type: method; for: Array.prototype
+urlPrefix: https://webassembly.github.io/spec/js-api/; spec: WASM-JS-API-1
+ type: interface
+  text: WebAssembly.Memory; url: #memory
+ type: attribute
+  text: buffer; for: WebAssembly.Memory; url: #dom-memory-buffer
```

Comment:
No idea if I'm doing this right. It *seems* to work fine? 😬

I'm also not sure why I even need to specify this manually. Am I missing some auto-linking magic that would allow me to just write `{{WebAssembly.Memory}}` and have it automatically find the correct spec (like `{{Response}}` [here](https://github.com/whatwg/streams/blob/0ff6d4508b58acce28386696b2854b2a1570d205/index.bs#L87))?

Reply:
I think the interface name should be `Memory`, so you'd do something like `{{Memory|WebAssembly.Memory}}`. However it appears not to work. I suspect that's because the Bikeshed database does not yet contain the spec: Maybe only when the AppHistory flag is enabled, I dunn

I'd suggest updating this to use `Memory` as the interface name, here and at the usage sites, so that in the future we can delete this link block.